model/system/metrics.yaml

```diff
@@ -430,3 +430,95 @@ groups:
       - ref: linux.memory.slab.state
     entity_associations:
       - host
+
+  - id: metric.system.linux.mdraid.disks
```

Comment:
Related to my above comment, these `mdraid` metrics can probably be moved to an `mdraid` model folder. This metric would be changed to just `mdraid.disks` and the same pattern would be applied to the others.

Reply:
Moved to a raid folder